### DISCUSSION TITLE
UTMStack: Bitdefender logs Integration issue

### DISCUSSION THREAD

**Discussion by agauttam** [utmstack/UTMStack#658] [Q&A] [answered]
While Integration of bitdefender I am facing the error -32602. Please help me to fix it

2024/04/30 10:17:57 BDGZ_Integration INFO: {"id":"1","jsonrpc":"2.0","error":{"code":-32602,"message":"Invalid params","data":{"details":"The web server with this URL must support TLS 1.2, at least"}}}
2024/04/30 10:18:12 BDGZ_Integration INFO: Checking configuration...
2024/04/30 10:18:12 BDGZ_Integration INFO: Status: %!(EXTRA string=200 OK)
2024/04/30 10:18:12 BDGZ_Integration INFO: {"id":"3","jsonrpc":"2.0","error":{"code":-32000,"message":"Server error","data":{"details":"Settings for event push service were not set"}}}

**Comment by Kbayero** [comment, marked as answer]
Hello @agauttam, this error is an error that Bitdefender returns when it cannot connect to the connector that receives the logs. Make sure you have port 8000 open on your firewall for the IPs mentioned in the integration guide

**Comment by agauttam** [comment]
Thanks issue is resolved now
